tensorbay/label/tests/test_subcatalog.py

```diff
@@ -0,0 +1,213 @@
+import pytest
+
+from ...utility import NameMixin, NameOrderedDict
+from .. import (
+    Box2DSubcatalog,
+    Box3DSubcatalog,
+    ClassificationSubcatalog,
+    Keypoints2DSubcatalog,
+    Polygon2DSubcatalog,
+    Polyline2DSubcatalog,
+    SentenceSubcatalog,
+)
+from ..attributes import AttributeInfo
+from ..supports import (
+    CategoryInfo,
+    KeypointsInfo,
+    SupportAttributes,
+    SupportCategories,
+    SupportIsTracking,
+)
+
+_DATA_CATEGORIES = {"categories": [{"name": "0"}, {"name": "1"}]}
+_DATA_ATTRIBUTES = {"attributes": [{"name": "gender", "enum": ["male", "female"]}]}
+_DATA_DELIMITER = {"categoryDelimiter": "-"}
+_DATA_TRACKING = {"isTracking": True}
+_DATA_KEYPOINTS = {
+    "keypoints": [
+        {
+            "number": 5,
+            "names": [
+                "L_shoulder",
+                "L_Elbow",
+                "L_wrist",
+                "R_Shoulder",
+                "R_Elbow",
+            ],
+            "skeleton": [(0, 1), (1, 2), (3, 4), (4, 5)],
+        }
+    ]
+}
+_LEXICON = ["齐", "q", "i2"]
+_DATA_SENTENCE = {"isSample": True, "sampleRate": 16000, "lexicon": [_LEXICON]}
+_COMMON_SUBCATALOG_CONTENT = dict(_DATA_TRACKING, **_DATA_CATEGORIES, **_DATA_ATTRIBUTES)
+_CLASSIFICATION_SUBCATALOG = dict(_DATA_CATEGORIES, **_DATA_ATTRIBUTES, **_DATA_DELIMITER)
+_KEYPOINTS2D_SUBCATALOG = dict(
+    _DATA_TRACKING, **_DATA_CATEGORIES, **_DATA_ATTRIBUTES, **_DATA_KEYPOINTS
+)
+_SENTENCE_SUBCATALOG = dict(_DATA_ATTRIBUTES, **_DATA_SENTENCE)
+
+
+@pytest.fixture
+def categories():
```

Comment:
why not use a constant?

Reply:
Pytest encourages use fixture to return complex test data, and I think packing this into fixtures is easier to manage